### PR TITLE
A simple database using typeorm

### PR DESCRIPTION
**Pull request by SeyedMojtaba1**

### CODE REVIEW

File: Seyed/TypeORM basic/dataSource.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import typeorm from 'typeorm'
+import { DataSource } from 'typeorm'
+import { User } from './entity'
+
+export const dataSource = new DataSource({
+    type: 'sqlite',
+    database: 'database.sqlite',
+    synchronize: true,
```
Comment (fatemeh-ra) [inline Seyed/TypeORM basic/dataSource.ts:8]: Remove the database and test the migrations with `synchronize: false`.

File: Seyed/TypeORM basic/index.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        seyed.age = 21
+    
+    await user.save(seyed)
+    console.log("User has been saved");
+
+    const users = await user.find()
+    console.log(users)
+}).catch(error => console.log(error));
+
+export default dataSource
```
Comment (fatemeh-ra) [inline Seyed/TypeORM basic/index.ts:20]: Why?

**Review by fatemeh-ra — CHANGES_REQUESTED** [review]

File: Seyed/TypeORM basic/dataSource.ts (hunk 1 above)
Comment (SeyedMojtaba1) [reply Seyed/TypeORM basic/dataSource.ts:8]: Ok sure.

File: Seyed/TypeORM basic/index.ts (hunk 2 above)
Comment (SeyedMojtaba1) [reply Seyed/TypeORM basic/index.ts:25]: I removed it.

Comment (fatemeh-ra) [reply Seyed/TypeORM basic/index.ts:25]: not removed

**Pull request merged**